samples/test/writeClient.js

```diff
@@ -0,0 +1,183 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ** This file is automatically generated by gapic-generator-typescript. **
+// ** https://github.com/googleapis/gapic-generator-typescript **
+// ** All changes to this file may be overwritten. **
+
+'use strict';
+
+const {assert} = require('chai');
+const {describe, it, before, after} = require('mocha');
+const uuid = require('uuid');
+const cp = require('child_process');
+const {BigQuery} = require('@google-cloud/bigquery');
+
+const bigquery = new BigQuery();
+
+const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
+
+const GCLOUD_TESTS_PREFIX = 'nodejs_bqstorage_samples_test';
+
+const generateUuid = () =>
+  `${GCLOUD_TESTS_PREFIX}_${uuid.v4()}`.replace(/-/gi, '_');
+
+const datasetId = generateUuid();
+
+describe('writeClient', () => {
+  let projectId;
+
+  before(async () => {
+    await deleteDatasets();
+
+    await bigquery.createDataset(datasetId);
```

Comment:
Spoke offline about this. Ideally we test in the default location (US) as well as in a non-US location. We do that in the Python tests.

https://github.com/googleapis/python-bigquery-storage/blob/81c8b8cdcc43d5ab671d7da7e8fe6c58291e1d49/samples/snippets/append_rows_pending_test.py#L38-L40

I'm fine with doing that in a follow-up PR, though.